### ISSUE TITLE
plasma turbines are unrealistic

### ISSUE THREAD

**Issue by bHXDgiuMa5yLHyC8** [GregTech6-Unofficial/GregTech6-Unofficial#8]
When Greg replaced GT5 fusion mechanic with GT6, it was arguably more realistic because you could pretend that the GT6 fusion reactor was doing some kind of direct energy conversion. I think that replacing it with a turbine that outputs RU is moving things back to being unrealistic. My suggestion is that either you rename the "plasma turbine" to "plasma MHD generator" and have it directly output EU instead of RU, or implement a lithium (or lithium+U238) blanket system that would be used with the heat exchanger.

**Comment by yuesha-yc** [comment]
Yeah indeed this is worth considering. Considering the most popular three methods: Li Blanket, U Blanket, and DEC (Direct-Energy-Conversion), I would probably think about how to make Fusion power more reasonable. I removed the Greg way of doing it because fusion reactor indeed needs energy input to maintain the magnetic field.
